### ISSUE TITLE
OSP Support: Changes to Infra Mapping Wizard - Step 3

### ISSUE THREAD

**Issue by vconzola** [ManageIQ/miq_v2v_ui_plugin#507]
Clearer instructional text

![5a-createmapping-datastores_2](https://user-images.githubusercontent.com/22823035/42962269-3434de02-8b5f-11e8-9109-6a56850f3a11.png)

For storage mapping for OSP, map VMWare datastores to OSP volume types

![5b-createmapping-datastores_2](https://user-images.githubusercontent.com/22823035/42962281-3cd7b232-8b5f-11e8-83a3-87b1e8defc2f.png)

**Comment by michaelkro** [comment]
@vconzola I have a question about the label for the Target Storages panel.  The mockup seems to suggest that for VMW -> OSP migrations the list items should be of the form, 

```
target_project.name \ target_project.volume_type (btw, I'm not sure if this attribute exists)
```

an example could be `ocp-registry \ <SOME_SORT_OF_VOLUME_TYPE>`

On the other hand, for VMW -> RHV migrations, the list items are

```
target_provider.name \ datastore.name
```

an example could be `RHV \ Export`

Is this discrepancy intentional?

**Comment by vconzola** [comment]
When Brett and I met with Marek from the OpenStack team he said that VMW datastores should map to OSP volume types. didn't make sense to me either. I think we need to ask the CF team for a sample API that shows what data we'll be receiving when OSP is selected as the target. We can either wait until next week or if you don't mind firing off a note to Adam and Greg B. they can probably respond sooner. I could send it but probably wouldn't use the right terms and just confuse everything.

**Comment by michaelkro** [comment]
@vconzola I'm drafting an email to Marek, Adam, and Greg  (I've cc'ed you)

**Comment by michaelkro** [comment]
Closed by #531
